testing/src/cypress/e2e/02-institution/04-manage-location.cy.ts

```diff
@@ -30,34 +40,139 @@ function loginAndClickOnAddNewLocation() {
   institutionManageLocationObject.addLocationButton().click();
 }
 
-function verifyLocationAndContactDetails() {
-  institutionManageLocationObject.institutionCode().should("be.visible");
-  institutionManageLocationObject.address1Text().should("be.visible");
-  institutionManageLocationObject.address2Text().should("be.visible");
-  institutionManageLocationObject.countryText().should("be.visible");
-  institutionManageLocationObject.cityText().should("be.visible");
+function verifyThatFieldShouldNotBeEmpty(
+  element: Cypress.Chainable<JQuery<HTMLElement>>,
+  errorMessage: Cypress.Chainable<undefined>
+) {
+  element.clear();
+  institutionManageLocationObject.submitButton().click();
+  errorMessage.should("be.visible");
+}
+
+function verifyThatElementIsVisibleAndDisabled(
+  element: Cypress.Chainable<JQuery<HTMLElement>>
+) {
+  element.should("be.visible").should("have.attr", "disabled", "disabled");
+}
+
+function verifyThatElementIsVisibleAndEnabled(
+  element: Cypress.Chainable<JQuery<HTMLElement>>
+) {
+  element.should("be.visible").should("not.be.disabled");
+}
+
+function verifyThatFieldDoesNotAcceptMoreThan100Chars(
```

Comment:
IMO the number of characters to be validated could be an argument to this util. And passed based on the field that is being validated.

Reply:
Updated it to be parameterized.